src/Illuminate/Database/Console/Factories/stubs/factory.stub

```diff
@@ -1,6 +1,9 @@
 <?php
 
+/* @var $factory \Illuminate\Database\Eloquent\Factory */
```

Comment:
Please remove this DocBlock.  #24647.

Reply:
Is this correct? I think DockBlocks have to start with `/**` and this is just `/*`

http://docs.phpdoc.org/guides/docblocks.html

Reply:
Such var description should be recognized correctly by any modern IDE. It is recognized by PHPStorm at least.
In the same time having this as PHPDoc block (e.g. `/**``) may cause problem with PHP code style fixers as they consider this to be a PHPDoc, which is not bound to function or class definition and thus is incorrect.